DirectXTex/DirectXTexP.h

```diff
@@ -199,6 +199,9 @@ using WICPixelFormatGUID = GUID;
 // HRESULT_FROM_WIN32(ERROR_NOT_SUPPORTED)
 #define HRESULT_E_NOT_SUPPORTED static_cast<HRESULT>(0x80070032L)
 
+// HRESULT_FROM_WIN32(ERROR_CANCELLED)
```

Comment:
Instead of using ``HRESULT_FROM_WIN32(ERROR_CANCELLED)``, just use ``E_ABORT``.

Reply:
Done.